### PR TITLE
Update the css-properties-values-api IDL file

### PR DESCRIPTION
**Pull request by lukebjerring**
Hello reviewer(s),

This PR is intended to consolidate the spec’s IDL definition with the WPT test suite’s copy, and any idlharness tests.

The up-to-date copy of the IDL file was automatically extracted from the reffy-reports repo (https://github.com/tidoust/reffy-reports/tree/master/whatwg/idl) which scrapes known specs automatically + regulary.

This PR is part of a migration project which will eventually be automatically updating and creating PRs for changes in spec IDL.

Please check that:
The Spec (and its source) is correct and up-to-date
All tests which cover the IDL in the spec have been migrated to fetch + use the idl in the `interfaces/` directory (instead of inline copies in the test files).

### CODE REVIEW

**Comment by w3c-bots** [comment]
# Build PASSED

Started: 2018-03-02 22:32:05
Finished: 2018-03-02 22:41:58


<h2>Failing Jobs</h2>
<ul>

<li>safari:11.0</li>

<li>MicrosoftEdge:16.16299</li>

</ul>




View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/25200)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/348432487)

File: interfaces/css-properties-values-api.idl [hunk 1]
```diff
@@ -0,0 +1,15 @@
+// GENERATED CONTENT - DO NOT EDIT
+// Content of this file was automatically extracted from the
+// "CSS Properties and Values API Level 1" spec.
+// See: https://drafts.css-houdini.org/css-properties-values-api-1/
+
+dictionary PropertyDescriptor {
+  required DOMString name;
+           DOMString syntax       = "*";
+           boolean   inherits     = false;
```
Comment (foolip) [inline interfaces/css-properties-values-api.idl]: @lukebjerring this is out of sync with the spec, can you update this PR?

**Review by foolip — CHANGES_REQUESTED** [review]

**Comment by lukebjerring** [comment]
Updated. PTAL.

**Comment by lukebjerring** [comment]
Note that we'll have an unimportant failure for the partial CSS namespace until #11819 lands.

**Review by foolip — APPROVED** [review]

**Pull request merged**